models/action.go

```diff
@@ -71,20 +71,21 @@ func init() {
 // repository. It implemented interface base.Actioner so that can be
 // used in template render.
 type Action struct {
-	ID          int64 `xorm:"pk autoincr"`
-	UserID      int64 `xorm:"INDEX"` // Receiver user id.
-	OpType      ActionType
-	ActUserID   int64       `xorm:"INDEX"` // Action user id.
-	ActUser     *User       `xorm:"-"`
-	RepoID      int64       `xorm:"INDEX"`
-	Repo        *Repository `xorm:"-"`
-	CommentID   int64       `xorm:"INDEX"`
-	Comment     *Comment    `xorm:"-"`
-	RefName     string
-	IsPrivate   bool      `xorm:"INDEX NOT NULL DEFAULT false"`
-	Content     string    `xorm:"TEXT"`
-	Created     time.Time `xorm:"-"`
-	CreatedUnix int64     `xorm:"INDEX"`
+	ID             int64 `xorm:"pk autoincr"`
+	UserID         int64 `xorm:"INDEX"` // Receiver user id.
+	OpType         ActionType
+	ActUserID      int64       `xorm:"INDEX"` // Action user id.
+	ActUser        *User       `xorm:"-"`
+	RepoID         int64       `xorm:"INDEX"`
+	Repo           *Repository `xorm:"-"`
+	CommentID      int64       `xorm:"INDEX"`
+	Comment        *Comment    `xorm:"-"`
+	CommentDeleted bool        `xorm:"default 0 not null"`
```

Comment:
Do not use 0 and 1 but true and false just like it is used in IsPrivate. Cant this field be reused for other optypes also?  So maybe is better call it IsDeleted?